pkg/planner/core/planbuilder.go

```diff
@@ -5160,9 +5162,10 @@ func buildShowSchema(s *ast.ShowStmt, isView bool, isSequence bool) (schema *exp
 	case ast.ShowVariables, ast.ShowStatus:
 		names = []string{"Variable_name", "Value"}
 	case ast.ShowCollation:
-		names = []string{"Collation", "Charset", "Id", "Default", "Compiled", "Sortlen"}
+		names = []string{"Collation", "Charset", "Id", "Default", "Compiled", "Sortlen", "Pad_attribute"}
 		ftypes = []byte{mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeLonglong,
-			mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeLonglong}
+			mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeLonglong, mysql.TypeVarchar}
+		flags = []uint{0, 0, mysql.UnsignedFlag | mysql.NotNullFlag, 0, 0, 0, 0}
```

Comment:
Seems `UnsignedFlag` will cause JDBC-5.x startup failed.

Reply:
@Defined2014 Do you have more info like versions and a stacktrace? Maybe a testcase I can run? Might be good to create a new issue for this?

Reply:
And as JDBC 4.3 is the latest JDBC version, I assume you meant MySQL Connector/J 5.x ?

Reply:
Yes, MySQL Connector/J 5.x. I only have a Chinese document, maybe you can use a translator to read it. https://www.cnblogs.com/youzhibing/p/18104262

Reply:
The translation feature in Firefox seems to do a good job on that document. Thanks.

Reply:
off-topic but it's interesting they have to dig deep into the Java code to learn something that is [clearly documented](https://dev.mysql.com/doc/connectors/en/connector-j-reference-type-conversions.html) :eyes:

| MySQL Type Name | Return value of `GetColumnTypeName` | Return value of `GetColumnClassName` |
|---|---|---|
| `BIGINT[(M)]` | `BIGINT` | `java.lang.Long` |
| `BIGINT[(M)] UNSIGNED` | `BIGINT UNSIGNED` | `java.math.BigInteger` |